docs/source/conf.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'eox-tagging'
+copyright = u'2020, eduNEXT'
+author = u'eduNEXT'
+
+# The short X.Y version
+version = u'0.3'
```

Comment:
Do we have to update this in each release?
 Could you change this to take the version from eox_tagging/__init__.py ? 
Or could you update bumpversion configuration to update this?

Reply:
Yes! I'll change it